onnxscript/utils/replace.py

```diff
@@ -9,19 +9,17 @@
 import onnx_ir.passes.common as common_passes
 
 
-def replace_functions(
-    model: onnx.ModelProto, functions: Sequence[onnx.FunctionProto]
-) -> onnx.ModelProto:
+def replace_functions_inplace(irmodel: ir.Model, irfunctions: Sequence[ir.Function]) -> None:
```

Comment:
```suggestion
def replace_functions_inplace(model: ir.Model, functions: Collection[ir.Function]) -> None:
```

Consider consistent names